assets/pages/paper-subscription-landing/components/content/deliveryTab.jsx

```diff
@@ -30,13 +30,12 @@ const ContentDeliveryFaqBlock = ({ setTabAction }: {setTabAction: typeof setTab}
     <ProductPageTextBlock title="How home delivery works">
       <p>
           If you live in Greater London (within the M25), you
-          can use The Guardian’s home delivery service. Don’t
-          worry if you live outside this area you can
+          can use The Guardian’s home delivery service. If don't, you can
           still <LinkTo tab="collection" setTabAction={setTabAction} >subscribe using our voucher scheme</LinkTo>.
       </p>
       <OrderedList items={[
         'Select your subscription below and checkout',
-        'Your subscribing deliveries will begin as quickly as five days from you subscribing',
+        'You'll receive your first newspaper as quickly as five days from you subscribing',
```

Comment:
`'You\'ll receive your first newspaper as quickly as five days from you subscribing',`
should do it 

Since the apostrophe is used to denote a string, if you want to put an apostrophe in the string, you need to use `\'` to make it clear that it's part of your text, and not the end of the string.